SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -222,6 +222,23 @@ JUPYTER_PATH=${ENV_PATH}/share/jupyter python -m ipykernel install --name "${ENV
 # We modify the already existing Python3 kernel with the kernel.json of the environment
 ln -f -s ${ENV_PATH}/share/jupyter/kernels/${ENV_NAME}/kernel.json /home/$USER/.local/share/jupyter/kernels/python3/kernel.json | tee -a ${LOG_FILE}
 
+# Pass Spark configurations to the environment kernel so it can be used in the notebooks
+if [ -n "${INSTALL_NXCALS}" ]; then
+    export SPARK_HOME="${ENV_PATH}/nxcals-bundle"
+    export PYSPARK_PYTHON="./environment/bin/python"
+    echo -e "export SPARK_HOME=\"${SPARK_HOME}\"\nexport PYSPARK_PYTHON=\"${PYSPARK_PYTHON}\"" >> /home/$USER/.bash_profile
+
+    # Passing configs to the kernel.json
+    export KERNEL_JSON=${ENV_PATH}/share/jupyter/kernels/${ENV_NAME}/kernel.json
```

Comment:
You don't need to export this variable, do you?

Reply:
See my comment above, to relocate this variable definition.

Reply:
No, we don't. Fixing it